packages/perseus/src/widgets/interactive-graphs/utils.test.ts

```diff
@@ -65,3 +65,58 @@ describe("normalizeCoords", () => {
         expect(result).toEqual(expected);
     });
 });
+
+describe("replaceOutsideTeX", () => {
+    test("no $s", () => {
```

Comment:
Since these tests do the same thing, using a table might make things easier since you won't need to redo the code for each test, just have a table of inputs
https://jestjs.io/docs/api#2-testeachtablename-fn-timeout

Reply:
I thought about that, but I wanted the description of each test to be explicit!

Reply:
Also I think the table would get really long with some of the descriptions + math strings. It would definitely be fewer lines, but I think I prefer the deliberateness of having them be separate tests in this particular case.